lib/src/main/java/dataframe/Row.java

```diff
@@ -161,6 +164,18 @@ public boolean equals(Object o) {
         });
     }
 
+    public int compareOnColumn(Row row, String column){
+        var columnType = this.schema.get(column).dataType().getSimpleName();
+        return switch(columnType){
```

Comment:
We can use the switch case on dataType directly instead of simpleName

Reply:
switch case on instances is not supported directly in Java 17. Marking this as resolved.